README.md

```diff
@@ -1,76 +1,65 @@
-git submodule as per the instructions below. To build, run:
+- **Official Documentation**: [jqlang.github.io/jq](https://jqlang.github.io/jq)
+- **Try jq Online**: [jqplay.org](https://jqplay.org)
 
-    git submodule update --init # if building from git to get oniguruma
-    autoreconf -i               # if building from git
-    ./configure --with-oniguruma=builtin
-    make -j8
-    make check
+## Installation
 
-Developers must `--enable-maintainer-mode` when making changes to the
-jq parser and lexer which also requires bison and flex to be installed.
+### Prebuilt Binaries
 
-To build a statically linked version of jq, run:
+Download the latest releases from the [GitHub release page](https://github.com/jqlang/jq/releases).
 
-    make LDFLAGS=-all-static
+### Docker Image
 
-After make finishes, you'll be able to use `./jq`.  You can also
-install it using:
+Pull the [jq image](https://github.com/jqlang/jq/pkgs/container/jq) to start quickly with Docker.
 
-    sudo make install
+### Building from source
 
-If you're not using the latest git version but instead building a
-released tarball (available on the website), then you won't need to
-run `autoreconf` (and shouldn't), and you won't need flex or bison.
+#### Dependencies
 
-To cross-compile for OS X and Windows, see docs/Rakefile's build task
-and scripts/crosscompile.  You'll need a cross-compilation environment,
-such as Mingw for cross-compiling for Windows.
+- libtool
+- make
+- automake
+- autoconf
 
-Cross-compilation requires a clean workspace, then:
+#### Instructions
 
-    # git clean ...
-    autoreconf -i
-    ./configure
-    make distclean
-    scripts/crosscompile <name-of-build> <configure-options>
+```console
+git submodule update --init # if building from git to get oniguruma
+autoreconf -i               # if building from git
+./configure --with-oniguruma=builtin
+make -j8
+make check
+sudo make install
+```
 
-Use the `--host=` and `--target=` ./configure options to select a
-cross-compilation environment.  See also
-["Cross compilation"](https://github.com/jqlang/jq/wiki/Cross-compilation) on
-the wiki.
+Build a statically linked version:
 
-To compile jq to WebAssembly, install the [Emscripten SDK](https://emscripten.org/docs/getting_started/downloads.html), then:
+```console
+make LDFLAGS=-all-static
+```
 
-    git submodule update --init # if building from git to get oniguruma
-    autoreconf -i               # if building from git
-    emconfigure ./configure --with-oniguruma=builtin
-    emmake make EXEEXT=.js CFLAGS="-O2" LDFLAGS="-s EXPORTED_RUNTIME_METHODS=['callMain']"
+To build from a released tarball (available on the release page), skip the `autoreconf` step, and flex or bison won't be needed.
 
+##### Cross-Compilation
 
-# Community
+For details on cross-compliation, check out the [GitHub Actions file](.github/workflows/ci.yml) and the [cross-compliation wiki page](https://github.com/jqlang/jq/wiki/Cross-compilation).
```

Comment:
Oh, I see, right, you're linking to the wiki pages.

Reply:
Yes, the cross-compilation details were moved to the wiki page. I'm trying to make the README more terse and include links for more details.